javascriptApi/examples.md

```diff
@@ -47,28 +44,26 @@ The code above will open a cobrowsing frame to `https://example.com`.
 ### Custom button
 We provide a convenient `Surfly.button()` function for rendering a nice default button. But it really doesn't do much, and it is very easy to make a custom button:
 
-```html
-<button class="my-custom-button">Start cobrowsing!</button>
+```javascript
+<button class="my-custom-button" style="display: none">Start cobrowsing!</button>
 
 <script>
-  var myBut = document.querySelector('.my-custom-button');
+  var myBtn = document.getElementById('my-custom-button');
```

Comment:
this will not work: "my-custom-button" is a class, not id. `querySelector()` is better here, I think you can just leave it as it was